### DISCUSSION TITLE
Support, issues or concerns re: lingui and Gatsby?

### DISCUSSION THREAD

**Discussion by francis-dion** [lingui/js-lingui#1106] [Q&A]
Hi, I'm developing an app and, based on web reviews, I think linguiJS is a slam dunk.
I'm also pretty much decided on building my web site on Gatsby, and possibly serving my app from it too.
I've seen the linguiJS starter pluggin/theme for Gatsby, but haven't seen much posts/community engagement around this integration. The few I found date back to 2018.
So my questions are:
1. Is linguiJS actively supported for Gatsby?
2. Are there any issues, concerns or gotchas I should be worried/know about?

Any help/resources you can point me to would also be greatly appreciated.

Thanks!
Francis

**Comment by semoal** [comment]
LinguiJs is actively maintained and supported for almost any framework out there that uses React.

I don't have experience with Lingui and Gatsby but I'll be happy to offer guidance or fixing issues you find with your Lingui + Gatsby integration

**Reply by francis-dion** [reply]
Hi @semoal . I went through the rabbit hole and am now thinking of dropping LinguiJS (which I really like) to get out of it.
I use gatsby-theme-i18n-lingui.
It depends on gatsby-theme-i18n.
Which adds <link rel="alternate" /> tags to <head>.
Which doesn't support localized urls (mysite.com/en/hello relates to mysite.com/fr/bonjour).
And I haven't found a fix/workaround with react-helmet.
In other words, I can't seem to be able to clean up link rel=alternate with the above stack, and that's a big problem for seo.
Any help would be appreciated, even though my feeling is this is not a problem with LinguiJS but with the dependencies of its Gatsby integration.
